content/en/docs/reference/kubernetes-api/labels-annotations-taints.md

```diff
@@ -114,3 +116,151 @@ The scheduler (through the _VolumeZonePredicate_ predicate) also will ensure tha
+
+Example: `endpointslice.kubernetes.io/skip-mirror="true"`
+
+Used on: Endpoint
+
+The label can be set to true on an Endpoint resource to indicate that the EndpointSliceMirroring controller should not mirror this resource with EndpointSlices.
+
+See [endpointslice.kubernetes.io/skip-mirror](#endpointslicekubernetesioskip-mirror).
+
+## service.kubernetes.io/service-proxy-name {#servicekubernetesioservice-proxy-name}
+
+Example: `service.kubernetes.io/service-proxy-name="foo-bar"`
+
+Used on: Service
+
+The label indicates that a custom service proxy will implement this service, kube-proxy does not generate iptables for the service.
+
+See [service.kubernetes.io/service-proxy-name](#servicekubernetesioservice-proxy-name).
+
+# Annotations
+
+## experimental.windows.kubernetes.io/isolation-type
+
+Example: `experimental.windows.kubernetes.io/isolation-type: "hyperv"`
+
+The annotation is used to run windows containers with hyperv isolation.
+
+At the moment this annotation only supports one container per pod.
+
+## ingressclass.kubernetes.io/is-default-class
+
+Example: `ingressclass.kubernetes.io/is-default-class: "true"`
+
+When a single IngressClass resource has this annotation set to true, new Ingress resource without a class specified will be assigned this default class.
+
+## kubernetes.io/ingress.class (deprecated)
+
+{{< note >}} Starting in v1.18, this annotation is deprecated in favor of [spec.ingressClassName]. {{< /note >}}
+
+## alpha.kubernetes.io/provided-node-ip
+
+Example: `alpha.kubernetes.io/provided-node-ip: "10.0.0.1"`
+
+The annotation is a node IP annotation set by the "external" cloud provider.
+
+When kubelet is started with the "external" cloud provider, then it sets this annotation on the node to denote an ip address set from the cmd line flag (node-ip). This ip is verified with the cloudprovider as valid by the cloud-controller-manager.
+
+# Taints
+
+## node.kubernetes.io/not-ready
+
+Example: `node.kubernetes.io/not-ready:NoExecute`
+
+The taint will be added when node is not ready and removed when node becomes ready.
+
+## node.kubernetes.io/unreachable
+
+Example: `node.kubernetes.io/unreachable:NoExecute`
+
+The taint will be added when node becomes unreachable (corresponding to NodeReady status Condition Unknown) and removed when node becomes reachable (NodeReady status Condition True).
+
+## node.kubernetes.io/unschedulable
+
+Example: `node.kubernetes.io/unschedulable:NoSchedule`
+
+The taint will be added when node becomes unschedulable and removed when node becomes scheduable.
+
+## node.kubernetes.io/memory-pressure
+
+Example: `node.kubernetes.io/memory-pressure:NoSchedule`
+
+The taint will be added when node has memory pressure and removed when node has enough memory.
+
+## node.kubernetes.io/disk-pressure
+
+Example: `node.kubernetes.io/disk-pressure:NoSchedule`
+
+The taint will be added when node has disk pressure and removed when node has enough disk.
```

Comment:
Maybe add "The kubelet detects disk pressure by observing the `imagefs.available`,
`imagefs.inodesFree`, `nodefs.available`, and `nodefs.inodesFree`(Linux only) metrics for a Node. The observed values are then compared to the corresponding thresholds that can be set on the kubelet to determine if the node condition and taint should be added/removed."